evasquez26/angular4-startup
# S1 one 160 Home page layout and styles partially applied

## Story / Task

[ONE-160](https://bixlabs.atlassian.net/browse/ONE-160)

## What this PR does?

Adds layout and styles for home components. Adds the right panel with the social links on the footer.


### Before 

Styles on the navbar, icons and right panel was missing.

### After

- Added styles to main page
- Added a right panel which will contain the social links

### How to test / reproduce

- Run the app
- Observe that most of the styles have been applied to the site

### TODOs

N/A

### Related PRs

N/A

## Migrations

NO

Thanks!
